client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -68,6 +86,58 @@ export default React.createClass( {
 		return ( this.props.domains || [ this.props.domain ] ).filter( domain => domain.currentUserCanManage );
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.pointsToWpcom );
+
+		debug( 'Domains:', this.getDomains() );
```

Comment:
The line is not exactly necessary, though there won't be a measurable performance impact either way. Regardless though, I thought `debug` statements are removed in production builds.